src/javascript/embed-api/components/view-selector2.js

```diff
@@ -235,6 +235,7 @@ gapi.analytics.ready(function() {
         account: this.account,
         property: this.property,
         view: this.view,
+        viewId: this.view.id,
```

Comment:
I think I was just mirroring your convenience variable `ids`. I have removed it.